### PR TITLE
Doc for Proxy migration

### PR DESCRIPTION
**Pull request by rohit-dimagi**
##### SUMMARY

* Doc for Proxy migration

##### ENVIRONMENTS AFFECTED

* ALL
##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: docs/howto/migrate-proxy.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#### Migrating proxy
+For Migrating Proxy we have to perform some manual steps as well. This document will list all the manual as well as automated process.
+1. Add the server in inventory and assign it the `proxy ` group and `cas_proxy`(For ICDS)
+2. Run  `commcare-cloud <env> ap deploy_shared_dir.yml --tags=nfs --limit=shared_dir_host`
+3. Run `commcare-cloud <env> ansible-playbook letsencrypt_cert.yml`
+4. Run `deploy-stack` for the server.
+5. Do a deploy `cchq <env> deploy`
+3. Copy static content from live proxy `/home/cchq/www/<env>/current/staticfiles` to new server
+4. Copy Letsencrypt Configuration dir from live proxy `/etc/letsencrypt` to new server
+5. Perfrom a dry run for cert check ` certbot renew --dry-run`
```
Comment (snopoke) [inline docs/howto/migrate-proxy.md]: why is it useful to run this?

Comment (rohit-dimagi) [reply docs/howto/migrate-proxy.md]: For checking any issue with Letsencrypt.

**Comment by rohit-dimagi** [comment]
Updated as suggested.

File: docs/howto/migrate-proxy.md (hunk 1 above)
Comment (snopoke) [reply docs/howto/migrate-proxy.md]: ok, but is it able to do that even though the traffic isn't being routed to this node yet?

Comment (rohit-dimagi) [reply docs/howto/migrate-proxy.md]: Moved to post validation section.

File: docs/services/nginx.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Nginx
+
+## Usage in CommCare
+
+[Nginx](https://nginx.org/en/docs/) nginx [engine x] is an HTTP and reverse proxy server. We use in Commcare for  Accelerated reverse proxying with caching,serving static files and load balancing;
```
Comment (snopoke) [inline docs/services/nginx.md]: > 
```suggestion
[Nginx](https://nginx.org/en/docs/) nginx [engine x] is an HTTP and reverse proxy server. It's primary use in CommCare is as a reverse proxy and load balancer however we also use it for serving static content, content caching and rate limiting.
```

**Pull request merged**